src/game/BoardState.java

```diff
@@ -29,41 +29,41 @@
 	public BoardState() {
 		this.pieces = new HashMap<Coordinate, AbstractPiece>();
 
-		this.pieces.put(new Coordinate(0, 0), new Cat(Owner.Player1));
-		this.pieces.put(new Coordinate(1, 0), new Dog(Owner.Player1));
-		this.pieces.put(new Coordinate(2, 0), new Horse(Owner.Player1));
-		this.pieces.put(new Coordinate(3, 0), new Camel(Owner.Player1));
-		this.pieces.put(new Coordinate(4, 0), new Elephant(Owner.Player1));
-		this.pieces.put(new Coordinate(5, 0), new Horse(Owner.Player1));
-		this.pieces.put(new Coordinate(6, 0), new Dog(Owner.Player1));
-		this.pieces.put(new Coordinate(7, 0), new Cat(Owner.Player1));
-
-		this.pieces.put(new Coordinate(0, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(1, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(2, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(3, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(4, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(5, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(6, 1), new Rabbit(Owner.Player1));
-		this.pieces.put(new Coordinate(7, 1), new Rabbit(Owner.Player1));
-
-		this.pieces.put(new Coordinate(0, 7), new Cat(Owner.Player2));
-		this.pieces.put(new Coordinate(1, 7), new Dog(Owner.Player2));
-		this.pieces.put(new Coordinate(2, 7), new Horse(Owner.Player2));
-		this.pieces.put(new Coordinate(3, 7), new Camel(Owner.Player2));
-		this.pieces.put(new Coordinate(4, 7), new Elephant(Owner.Player2));
-		this.pieces.put(new Coordinate(5, 7), new Horse(Owner.Player2));
-		this.pieces.put(new Coordinate(6, 7), new Dog(Owner.Player2));
-		this.pieces.put(new Coordinate(7, 7), new Cat(Owner.Player2));
-
-		this.pieces.put(new Coordinate(0, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(1, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(2, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(3, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(4, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(5, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(6, 6), new Rabbit(Owner.Player2));
-		this.pieces.put(new Coordinate(7, 6), new Rabbit(Owner.Player2));
+		this.pieces.put(new Coordinate(0, 0), new Cat(Owner.Player2));
```

Comment:
Just converted all the y values to be 7 - y. It turned out to more convenient to keep the axis in the same orientation.